roles/traefik/tasks/main.yml

```diff
@@ -8,6 +8,29 @@
     - traefik
     - bootstrap
 
+- name: create directory for local traefik service override
+  sudo: yes
+  file:
+    dest:  /etc/systemd/system/traefik.service.d
+    state: directory
+    mode: 0755
+  tags:
+    - traefik
+    - bootstrap
+
+- name: create local traefik service override
+  sudo: yes
+  copy:
+    dest: /etc/systemd/system/traefik.service.d/marathon.conf
+    content: |
+      [Service]
+      ExecStartPre=-/usr/bin/smlr http {{ traefik_marathon_endpoint }}/v2/leader
```

Comment:
I understand the convenience of having this inline, but most everything else we do like this is rendered as a template. Should this be as well for consistency's sake?